x-pack/plugins/security_solution/public/timelines/components/timeline/footer/index.tsx

```diff
@@ -174,30 +174,36 @@ export const EventsCount = React.memo(EventsCountComponent);
 
 EventsCount.displayName = 'EventsCount';
 
-export const PagingControlComponent = ({
-  activePage,
-  isLoading,
-  onPageClick,
-  totalPages,
-}: {
+interface PagingControlProps {
   activePage: number;
   isLoading: boolean;
   onPageClick: OnChangePage;
   totalPages: number;
-}) => (
-  <>
-    {isLoading ? (
-      `${i18n.LOADING}...`
-    ) : (
-      <EuiPagination
-        data-test-subj="timeline-pagination"
-        pageCount={totalPages}
-        activePage={activePage}
-        onPageClick={onPageClick}
-      />
-    )}
-  </>
-);
+}
+
+export const PagingControlComponent: React.FC<PagingControlProps> = ({
+  activePage,
+  isLoading,
+  onPageClick,
+  totalPages,
+}) => {
+  if (isLoading) {
+    return <>{`${i18n.LOADING}...`}</>;
+  }
+
+  if (!totalPages) {
```

Comment:
While desk testing this change, I found a bug where the paging controls disappear when the last page of results is clicked. To reproduce:

1. Drag anything to the timeline

2. Click on the last page of results

**Expected result**
- The last page of results is displayed

**Actual result**
- The paging controls disappear, per the following screenshot:

![last-page-of-results](https://user-images.githubusercontent.com/4459398/94630938-6518e100-0284-11eb-9589-13e3648a0905.gif)